.github/workflows/ci.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
   lint-ui:
     runs-on: ubuntu-latest
     container:
-      image: node:20.12.2
+      image: node:22.6.0
```

Comment:
While changing this, I wondered if there is a specific reason for it to rely on a Node image, which then again installs... Node.

Reply:
Feel free to amend this PR with the fix, but I am premptively approving.

Reply:
Picked to drop the container, as the Node action appeared to be more mature (and easier to set up) in terms of caching.